### DISCUSSION TITLE
How do I show the menu again?

### DISCUSSION THREAD

**Discussion by Alfpv** [kubilus1/autoortho#177] [Q&A]
This is a really awesome app. I ticked the option to hide the options menu on launch, but I can't seem to find  a way of switching it back on again - can anyone help?

**Comment by Simssmith** [comment]
close autoortho
open .autoortho file in your home directory (For example "c:\users\simssmith\ .autoortho" using notepad or any text editor. 

It should look like this under "general"

```
[general]
# use gui config at startup
gui = True
```
Save the file. 
Start AutoOrtho again.
